HTTPServerProject/Response.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Net;
+using System.Net.Sockets;
+using System.IO;
+using HTTPServerProject.WriteStreams;
+
+namespace HTTPServerProject.Responses
+{
+    public class Response
+    {
+        IWriteStreams writer = default!;
```

Comment:
You don't need these `default!!` declarations when declaring variables if your constructor initializes the variable (which it does below).

Reply:
Sounds good. I was able to refactor the code for this update.